src/common/src/session_config/mod.rs

```diff
@@ -152,7 +152,11 @@ pub struct SessionConfig {
     #[parameter(default = true, rename = "rw_streaming_enable_bushy_join")]
     streaming_enable_bushy_join: bool,
 
-    /// Enable arrangement backfill for streaming queries. Defaults to false.
+    /// Enable arrangement backfill for streaming queries. Defaults to true.
```

Comment:
Pardon. Defaults to true or false? IIRC, we want to make it false by default?

Reply:
Nope, true by default. It should only be false for cloud side.